components/dropbox/actions/download-file-to-tmp/download-file-to-tmp.mjs

```diff
@@ -0,0 +1,54 @@
+import dropbox from "../../dropbox.app.mjs";
+import common from "../common/common.mjs";
+import fs from "fs";
+import { file } from "tmp-promise";
+
+export default {
+  ...common,
+  name: "Download File To TMP",
+  description: "Download a specific file to the temporary directory. [See docs here](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesDownload__anchor)",
+  key: "dropbox-download-file-to-tmp",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    dropbox,
+    path: {
+      propDefinition: [
+        dropbox,
+        "pathFile",
+      ],
+      description: "The file path in the user's Dropbox to download.",
+    },
+    name: {
+      type: "string",
+      label: "File name",
+      description: "The new name of the file to be saved, including it's extension. e.g: `myFile.csv`",
+      optional: true,
+    },
+  },
+  async run({ $ }) {
+    const { result } = await this.dropbox.downloadFile({
+      path: this.getNormalizedPath(this.path, false),
+    });
+
+    const {
+      path, cleanup,
+    } = await file();
+
+    const tmpPath = this.name
+      ? `/tmp/${this.name}`
+      : path;
+
+    await fs.promises.appendFile(tmpPath, Buffer.from(result.fileBinary));
+    await cleanup();
+
+    delete result.fileBinary;
+
+    $.export("$summary", `File successfully saved in "/tmp/${this.name}"`);
```

Comment:
I believe that this export summary is wrong when the user doesn't provide the prop name

Reply:
I believe that it should be `$.export("$summary", `File successfully saved in "${tmpPath}"`);`